### PR TITLE
Update the `bal` Command in BBEs

### PR DESCRIPTION
**Pull request by praneesha**
## Purpose
Update the `bal` command in BBEs.

## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 [![Ballerina Distribution Build](https://github.com/ballerina-platform/ballerina-distribution/workflows/Ballerina%20Distribution%20Build/badge.svg)](https://github.com/ballerina-platform/ballerina-distribution/actions?query=workflow%3A%22Ballerina+Distribution+Build%22)
 [![Daily build](https://github.com/ballerina-platform/ballerina-distribution/workflows/Daily%20build/badge.svg)](https://github.com/ballerina-platform/ballerina-distribution/actions?query=workflow%3A%22Daily+build%22)
 
-The Ballerina distribution repository builds the final Ballerina distributions. It combines the Ballerina runtime with standard libraries and language extensions.
+The Ballerina distribution repository builds the final Ballerina distributions. It combines the bal runtime with standard libraries and language extensions.
```
Comment (ThisaruGuruge) [inline README.md:5]: Do we need this change?

File: ballerina-test/src/test/resources/openapi/integration-tests/Ballerina 2.toml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+[package]
+org= "ballerina"
+name= "openAPI"
+version= "0.1.0"
```
Comment (ThisaruGuruge) [inline ballerina-test/src/test/resources/openapi/integration-tests/Ballerina 2.toml:4]: Extra file?

**Review by ThisaruGuruge — COMMENTED** [review]
There are lots of duplicated files in this PR. Please check.

**Review by ThisaruGuruge — CHANGES_REQUESTED** [review]
Shall we remove the duplicates?

**Comment by praneesha** [comment]
Closing since this is invalid.

**Pull request closed without merging**